### PR TITLE
refactor so “impact” can be one line

### PR DESCRIPTION
**Pull request by ultrasaurus**
need to go with current dashboard feature set
markdown (including line breaks!) is a future feature

### CODE REVIEW

**Comment by ultrasaurus** [comment]
@rogeruiz can you proof-read and merge if it looks good?

**Comment by ultrasaurus** [comment]
by the way, here's a preview using new dashboard feature that is not yet merged (partner section is missing cuz preview tool doesn't support it)

image

File: .about.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ description: >
   Open Opportunities is creating a network of federal employees that
   will lead to a more effective, efficient, responsive government by sharing skills
   and collaborating on projects that support the mission of an agency or the
-  government as a whole.
+  government as a whole.    Only 37% of federal employees surveyed believe that
```
Comment (rogeruiz) [inline .about.yml]: I'm going to clean up this whitespace between `whole.` and `Only`.

File: .about.yml [hunk 2]
```diff
@@ -18,7 +22,7 @@ partners:
 stage: live
 
 impact: >
-  65 agencies, 3000 subscribers\n\n278 projects completed, 40 in progress\n\ntotal market: 2.7 million federal employees\n\noutcome: skilled, efficient, agile workforce
+  Amplify the work of 1 Million federal employees.
```
Comment (rogeruiz) [inline .about.yml]: I'm also going to update down-case `Million` here as well.

**Comment by rogeruiz** [comment]
Once these builds pass, I'll merge this in and include it in `0.8.2`

**Pull request merged**